doc/admin-guide/plugins/combo_handler.en.rst

```diff
@@ -83,3 +83,16 @@ results in these file paths being "reconstructed"::
     /dir:path2/file5
     /dir:path2/file6
 
+Caching
+=======
+Combohandler follows a few rules for the "Cache-Control" header:
+
+1) All requested documents must have "immutable" for the combo'd
```

Comment:
Not that I knew this beforehand, but this is restructured text and it looks like this is the correct list syntax. Got lucky I guess.

https://en.wikipedia.org/wiki/ReStructuredText#Lists